packages/admin/src/pages/service-error/index.page.tsx

```diff
@@ -40,15 +43,19 @@ const ServiceErrorPage = ({
               <p className="govuk-body">Try again later.</p>
               <p className="govuk-body">{serviceError.errorInformation}</p>
               {serviceError.linkAttributes && (
-                <p className="govuk-body">
-                  <CustomLink
-                    href={serviceError.linkAttributes.href}
-                    excludeSubPath={excludeSubPath}
-                  >
-                    {serviceError.linkAttributes.linkText}
-                  </CustomLink>
-                  {serviceError.linkAttributes.linkInformation}
-                </p>
+                <>
+                  <p className="govuk-body">
+                    <CustomLink
+                      href={serviceError.linkAttributes.href}
+                      excludeSubPath={excludeSubPath}
+                    >
+                      {serviceError.linkAttributes.linkText}
+                    </CustomLink>
+                  </p>
+                  <p className="govuk-body">
+                    {serviceError.linkAttributes.linkInformation}
+                  </p>
+                </>
```

Comment:
separated a line into separate paragraphs as it was looking like

Please try again.Your progress has been saved